packages/server-ct/src/server-ct.ts

```diff
@@ -126,36 +125,37 @@ class Server {
     })
   }
 
-  _onRequest (headers, automationRequest, options) {
-    return this._request.sendPromise(headers, automationRequest, options)
-  }
-
   _callRequestListeners (server, listeners, req, res) {
     return listeners.map((listener) => {
       return listener.call(server, req, res)
     })
   }
 
+  async _destroyAsync () {
+    allowDestroy(this._server)
+
+    return promisify(this._server.close)()
+    .catch(() => { }) // dont catch any errors
+  }
+
   _close () {
     // bail early we dont have a server or we're not
     // currently listening
     if (!this._server || !this.isListening) {
       return Bluebird.resolve()
     }
 
-    return this._server.destroyAsync()
```

Comment:
Why are we no longer returning the value of this? Was it void?

Reply:
seems `_destroyAsync` never existed in the first place - but I added it (copy pasting) so I will put this back in